R/test-module.R

```diff
@@ -64,33 +64,55 @@ testModule <- function(module, expr, ...) {
   )
 }
 
+isOldModule <- function(func) {
```

Comment:
It would be helpful to have a comment explaining the purpose of this function (and how it works).